js-exercises/cache-function/cacheFunction.js

```diff
@@ -0,0 +1,15 @@
+const cacheFunction = (fn) => {
+  let cache = {}
```

Comment:
Objects in JS can only support strings and symbols as keys. A better choice would be a `Map`.
```suggestion
  let cache = new Map();
```

Reply:
Since you are not re-assigning the `cache`, you can make it `const`.
```suggestion
  const cache = {}
```